js/src/components/free-listings/configure-product-listings/checkErrors.js

```diff
@@ -27,15 +27,41 @@ const checkErrors = (
 
 	if (
 		values.shipping_rate === 'flat' &&
-		( shippingRates.length < finalCountryCodes.length ||
-			shippingRates.some( ( el ) => el.rate < 0 ) )
+		( values.shipping_country_rates.length < finalCountryCodes.length ||
+			values.shipping_country_rates.some( ( el ) => el.rate < 0 ) )
 	) {
 		errors.shipping_rate = __(
 			'Please specify shipping rates for all the countries. And the estimated shipping rate cannot be less than 0.',
 			'google-listings-and-ads'
 		);
 	}
 
+	/**
+	 * Check offer free shipping.
+	 */
+	if (
+		values.offer_free_shipping === undefined &&
+		values.shipping_country_rates.some( isNonFreeFlatShippingRate )
+	) {
+		errors.offer_free_shipping = __(
+			'Please select an option for offering free shipping.',
```

Comment:
Not sure if this meaning is a bit ambiguous. I'd like to confirm would it be better to describe with something like "Please select an option for whether to offer free shipping."?

Reply:
How is it ambiguous? 🤔 

Anyway, I have changed it based on your suggestion: [`dc8a6d9` (#1309)](https://github.com/woocommerce/google-listings-and-ads/pull/1309/commits/dc8a6d939cb542b8cf3e99f2c3c470dc19a2450c). I think that works. 😄

Reply:
Thanks!

> How is it ambiguous?

The message is used to prompt the merchant for choosing the Yes/No option for whether to offer free shipping. "Please select an option for offering free shipping." looks like asking the merchant to offer free shipping, but my English ability can't confirm if I got the meaning wrong. 🤦‍♂️  So I wanted to double-confirm it. 🙏